src/context.ts

```diff
@@ -21,12 +21,13 @@ export interface Inputs {
   configInline: string;
   append: string;
   cleanup: boolean;
+  keepState: boolean;
 }
 
 export async function getInputs(): Promise<Inputs> {
   return {
     version: core.getInput('version'),
-    name: await getBuilderName(core.getInput('driver') || 'docker-container'),
+    name: getBuilderName(core.getInput('name'), core.getInput('driver') || 'docker-container'),
```

Comment:
`name` needs to be defined in https://github.com/docker/setup-buildx-action/blob/master/action.yml. Also needs to update README to add this new input.